src/engine/game/world.lua

```diff
@@ -313,11 +313,32 @@ function World:registerCall(name, scene)
 end
 
 --- Replaces a phone call in the Light World CELL menu with another
----@param name  string          The name of the call as it will show in the CELL menu
----@param index integer         The index of the call to replace
----@param scene string          The cutscene to play when the call is selected
-function World:replaceCall(name, index, scene)
-    self.calls[index] = {name, scene}
+---@param replace_name string          The name of the call to replace
+---@param name         string          The name of the call as it will show in the CELL menu
+---@param scene        string          The cutscene to play when the call is selected
+function World:replaceCall(replace_name, name, scene)
+    for i,call in ipairs(self.calls) do
+        if call[1] == old_name then
```

Comment:
```suggestion
        if call[1] == replace_name then
```
Incorrect variable name here

Reply:
fixed